src/DiIiS-NA/D3-GameServer/GSSystem/GameSystem/GSBackend.cs

```diff
@@ -151,13 +151,13 @@ public void CheckQuestCriteria(ulong gameAccountId, int questId, bool isCoop)
 		public void CheckKillMonsterCriteria(ulong gameAccountId, int actorId, int type, bool isHardcore)
 		{
 			Logger.MethodTrace($"gameAccountId {gameAccountId}, actorId {actorId}, type {type}, hc {isHardcore}");
-			BattleNetSocketSend($"ckmc|{gameAccountId}/{actorId}/{type}/{(isHardcore ? "True" : "False")}";
+			BattleNetSocketSend($"ckmc|{gameAccountId}/{actorId}/{type}/{ (isHardcore ? "True" : "False")}");
```

Comment:
Is this new space (between '{' and '(') expected?

Reply:
Sorry, but I don't know. I just notice that missing brackets and put there to back the compilation.

Reply:
@rgto, I don't know too, but I don't see spaces in other places, so probably this extra space can break something